src/XeroPHP/Enums/Accounting/TaxRates/TaxTypes/Singapore.php

```diff
@@ -0,0 +1,47 @@
+<?php
```

Comment:
You think it's worth putting the rates here as a secondary const BLINPUT_RATE = 0